src/pages/authenticationPage/Signin.js

```diff
@@ -61,62 +57,102 @@ const Signin = () => {
-              name="password"
-              required
-              value={setUserInfo.password}
-              onChange={(event) =>
-                setUserInfo({ ...userInfo, password: event.target.value })
-              }
-            />
-            <div className="show-password" id="show-create-password">
-              show
+            if (!values.email) {
+              errors.email = "Required";
+            } else if (
+              !/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i.test(values.email)
+            ) {
+              errors.email = "Invalid email address";
+            }
+            return errors;
+          }}
+        >
+          <Form className="form">
+            <div className="login-heading">Login</div>
+            <div>
+              <label htmlFor="email" className="form-label">
+                Email
+              </label>
+              <Field
+                className="input"
+                type="text"
+                placeholder="Enter Email"
+                name="email"
+              />
+              <ErrorMessage
+                className="error-message"
+                name="email"
+                component="div"
+              />
             </div>
-          </div>
 
-          <div className="login-store flex flex-space-between">
-            <label htmlFor="store">
-              <input type="checkbox" />I accept all terms & conditions
-            </label>
-          </div>
+            <div>
+              <label htmlFor="password" className="form-label">
+                Password
+              </label>
+              <div className="show-hide-password">
+                <Field
+                  id="create-password"
+                  className="input"
+                  type={showPassword ? "text" : "password"}
+                  placeholder="Enter Password"
+                  name="password"
+                />
+                <div
+                  className="show-password"
+                  onClick={() => {
+                    setShowPassword(!showPassword);
+                  }}
+                >
+                  show
+                </div>
+              </div>
+              <ErrorMessage
+                className="error-message"
+                name="password"
+                component="div"
+              />
+            </div>
 
-          <div className="form-btn">
-            <button className="btn btn-primary btn-lg">Login</button>
-          </div>
-          <button className="btn btn-primary btn-lg" onClick={guestLogin}>
-            Guest Login
-          </button>
-          <div className="create-account flex flex-center">
-            <Link to="/signup">
-              <p>Don't have account? Create now</p>
-            </Link>
-          </div>
-        </form>
+            <div className="login-store flex flex-space-between">
+              <label htmlFor="store">
+                <input type="checkbox" />I accept all terms & conditions
```

Comment:
You can add the id & for attribute here so that when user clicks on the text, the checkbox happens, currently it is only happening on checkbox click

Reply:
yes we can do that but keeping as click on box only